OpenRA.Mods.Common/Activities/Air/FlyForward.cs

```diff
@@ -36,11 +45,14 @@ public override bool Tick(Actor self)
 				return true;
 			}
 
-			if (IsCanceling || remainingTicks-- == 0)
+			if (IsCanceling || (flyTicks != 0 && ticks++ >= flyTicks) || (flyTicks == 0 && remainingDistance <= 0))
```

Comment:
Can we have a comment somewhere saying that it is explicitly valid for `flyTicks` to be -1 to mean "keep flying until cancelled"?